manifests/sidecar-template.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sidecar-template
+  labels:
+    app.kubernetes.io/component: template
```

Comment:
I think we should also add the `chaos-mesh` label for this configmap

Reply:
Good suggestion!

Reply:
What is the specific label you mean? Is it?
```
    app.kubernetes.io/instance: chaos-mesh
    app.kubernetes.io/name: chaos-mesh
```